thorlcr/msort/tsortm.cpp

```diff
@@ -1422,9 +1431,9 @@ class CSortMaster : public IThorSorterMaster, public CSimpleInterface
                     char url[100];
                     slave.endpoint.getUrlStr(url,sizeof(url));
                     if (splitMapUpper)
-                        slave.MultiMergeBetween(numnodes*numnodes,splitMap,splitMapUpper,numnodes,endpoints);
+                        slave.MultiMergeBetween(max, numnodes*numnodes,splitMap,splitMapUpper,numnodes,endpoints);
```

Comment:
pedantic, but I think 'max' is the maximum number of records on any node - whereas 'total' (or possibly stotal) is the globalCount.  Either would work for the current use, but passing total would be clearer.